packages/lib-utils/src/components/table/VirtualizedTable.tsx

```diff
@@ -125,7 +125,9 @@ const VirtualizedTable: React.FC<VirtualizedTableProps> = ({
 
   const paginateData = (allData: AnyObject[]) => {
     const end =
-      pagination?.offset && pagination?.limit ? pagination.offset + pagination.limit : undefined;
+      typeof pagination?.offset === 'number' && pagination?.limit
```

Comment:
Why do we have to check if `pagination.offset` is really a number? Is this because `0` is falsy value?

Reply:
@vojtechszocs yeah, that's the reason